lib/org/chromium/webidl/IDLFileContents.js

```diff
@@ -19,7 +19,9 @@ foam.CLASS({
       class: 'FObjectProperty',
       of: 'org.chromium.webidl.IDLFile',
       name: 'metadata',
-      required: true,
+      factory: function() {
```

Comment:
@m-cheung why not `value: null`?

Reply:
An oversight on my part. It can be changed to `value: null`
Sorry about that.